### DISCUSSION TITLE
Row filter using FilterValue

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#6673] [Help]
Please I want to filter a lookup using 2 values but that's not possible.
Is there a way I can filter like SQL **IN** rather than **=**

`[LookupEditor(typeof(PurchasesRow), FilterField = "Status",FilterValue =  "1,2"),LookupInclude]`

I dont want to use the `override` option below as it will affect all other calls
`protected override void ApplyFilters(SqlQuery query)`

**Comment by arunkumar12712** [comment]
https://github.com/serenity-is/Serenity/wiki/Serenity-LookupEditor---Pass-Parameter-And-Change-LookupKey

**Comment by rhemasystems** [comment]
Thanks arunkumar for the link, but that solution is rather complex looking at what I want to achieve, I was able to use the below to achieve what I want, except that I want to use the `Serenity.Criteria.Operator.in` criteria, but how to pass in the values like SQL `IN (1,2)` is my issue.
Please assist

`Dialog.ts`

         protected afterLoadEntity() {
            super.afterLoadEntity();
            this.form.ReceiptLineItems.setItems([]);
            const fldp = SmartERP.Purchases.PurchasesRow;
            var criteria: any[]; this.form.PurchaseId.clearItems();
            SmartERP.Purchases.PurchasesService.List({
                Criteria:
                    Serenity.Criteria.and(
                        criteria,
                        [[fldp.Fields.Status], Serenity.Criteria.Operator.ne, 2]
                    )
            }, resp => {
                resp.Entities.forEach(element => {
                    var item: Serenity.Select2Item = {
                        id: element.Id.toString(),
                        text: element.InvoiceNo
                    }
                    this.form.PurchaseId.addItem(item)

                })
            })
        )`
